docs: explain how to override global automerge settings (#11099)
renovatebot/renovate: docs/usage/automerge-configuration.md

### Codeowners

Depending on the platform, having a `CODEOWNERS` file could block automerging, because it means a code owner must review the PR.

> ### Disabling automerge in select repos, rest of org should automerge

Author: I'm not sure if this `h3` title is the correct summary of the section below, it's a bit long right now.

If you have any ideas, please suggest them!

Reviewer: Maybe `### Overriding global automerge`

Author: ```suggestion
### Overriding global automerge
```

Let's go with that for now! 😄

> You might have setup a global configuration in a `.github` repository, that has a `renovate.json` file that turns on automerge for certain dependencies.

Reviewer 2: Shouldn't we tell this shared preset? Not all people are using `.github` repo for this. 😏

Author: I'm sorry, I really don't understand what you mean here. Can you try explaining again, but with different words?

Reviewer: I think we've done enough describing already. There's nothing about this overriding which is specific to automerge anyway.

Author: ```suggestion
You might have setup a global configuration in a `.github` repository, that has a `renovate.json` file that turns on automerge for certain dependencies.
It does not matter where you've put the global config, the important point in this example is that you're extending from a global config that's somewhere else.
For this example we'll assume you put your config in a repository on GitHub, called `.github`.
```

How about this then? It explains a bit more, and tells readers that the important point is that you're extending from a config that's _somewhere else_.

Repositories in the organization all extend from this global configuration, and so they "inherit" the automerge settings as well.

> To turn off automerge in full for select repositories, you need to make a config that overrides all `packageRules` in the repository's `renovate.json` file, like this:

Author: I'm not sure if this properly explains the use-case for the config below, so please suggest changes to the wording.

Reviewer 2: ```suggestion
You might have setup a global configuration in a `.github` repository, that has a `renovate.json` file that turns on automerge for certain dependencies.
Repositories in the organization all extend from this global configuration, and so they "inherit" the automerge settings as well.
To turn off automerge for all dependnecies of a selected repository, you need to make a config that overrides all `packageRules` in the repository's `renovate.json` file, like this:
```

Something like this?

Author: I think your text is a bit clearer than mine, so I'll apply your suggestion. 😄
